packages/app/src/components/add-tag/AddTagStepTrack.vue

```diff
@@ -110,6 +110,8 @@ export default defineComponent({
       const tagCopy = tag.value;
       tagCopy.spotifyTrackUri = track.uri;
       tagCopy.imageUrl = track.imageUri;
+      tagCopy.trackTitle = track.title;
+      tagCopy.trackArtists = track.artists.join(', ');
```

Comment:
Later we can (should change this)
In the PR #224 Implement UI for editing a tag I added a service to get the 

- title
- Artists

by the spotify uri